packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -51,6 +52,12 @@ class TestRenderingFlutterBinding extends BindingBase with Scheduler, Services,
     if (phase == EnginePhase.paint)
       return;
     renderView.compositeFrame();
+    if (phase == EnginePhase.composite)
```

Comment:
Yeah, there is some duplication of code between the rendering tests and the widget tests. I can file a bug to consolidate it.